openshift_images/mysql.adoc

```diff
@@ -0,0 +1,109 @@
+= MySQL
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Our OpenShift MySQL database image https://github.com/openshift/mysql[repository] contains the sources and Dockerfiles for building MySQL database images. The resulting images can be run by Docker.
+
+== Versions
+Current supported versions of MySQL:
+
+* https://github.com/openshift/mysql/tree/master/5.5[5.5]
+
+== Base Images
+
+* RHEL-7
+* CentOS-7
+
+[NOTE]
+====
+RHEL-7 image is not available as trusted build in https://index.docker.io[Docker Index]. In order to perform build or test action on a RHEL-7 based MySQL image, you need to run the build on properly subscribed RHEL-7 machine.
```

Comment:
nothing is available as a trusted build in docker hub anymore, so just say that it's not available on docker hub.